dojson/utils.py

```diff
@@ -30,7 +30,7 @@ def int_with_default(value, default):
 
 
 def ignore_value(f):
-    """Remove key for None value.
+    """Remove key for None or empty value value.
```

Comment:
What about empty strings?, sometimes you want to keep this as a "value" (no values is a value it self)